### ISSUE TITLE
svneverever --committers fails on commits without committer information

### ISSUE THREAD

**Issue by mandree** [hartwork/svneverever#2]
```
$ python2 svneverever --committers $MYREPO
Analyzing 7078 revisions...
100.00%  (00:00:00 remaining)  [###################################################################################################]

Traceback (most recent call last):
  File "svneverever", line 17, in <module>
    main()
  File "modules/svneverever/main.py", line 321, in main
    dump_nick_stats(nick_stats, digit_count(latest_revision), config=args)
  File "modules/svneverever/main.py", line 111, in dump_nick_stats
    print(format % (commit_count, first_commit_rev, last_commit_rev, _encode(nick)))
  File "modules/svneverever/main.py", line 61, in _encode
    return text.encode(sys.stdout.encoding or 'UTF-8', 'replace')
AttributeError: 'NoneType' object has no attribute 'encode'

```

**Comment by hartwork** [comment]
Hi!

I believe the problem is not with encoding. it looks like you have a commit without any author and svneverever seems not ready to handle that.

Could you try again with this patch applied?
[svneverever-issue-2.patch.txt](https://github.com/hartwork/svneverever/files/3194049/svneverever-issue-2.patch.txt)

**Comment by mandree** [comment]
The patch helps on my repo:
```
$ svneverever --committers $HOME/VCS-mine/bogofilter.svnroot
Analyzing 7079 revisions...
100.00%  (00:00:00 remaining)  [###################################################################################################]

 289 (   1; 6626)  <author missing>
 [...]
2270 (  19; 7079)  mandree
[...]
```

**Comment by hartwork** [comment]
Thanks for testing the patch!

**Comment by hartwork** [comment]
Release 1.3.1 on PyPI now contains [this fix](41760a11a7fd48c61407b7426f40929d902ab8a2). Closing as fixed.
